src/arp.c

```diff
@@ -70,6 +71,14 @@ int arp_cache_insert(in_addr_t ip_addr,
     if (ip_addr == 0)
         return 0;
 
+    if ((entry = arp_cache_get_entry(ip_addr))>0) {
```

Comment:
Run `clang-format -i` to ensure the consistent style.

Reply:
I ensured the consistent style.